src/deploy/functions/backend.ts

```diff
@@ -196,6 +212,7 @@ export interface Backend {
   schedules: ScheduleSpec[];
   topics: PubSubSpec[];
   environmentVariables: EnvironmentVariables;
+  endpoints: Endpoint[];
```

Comment:
Should this be an array or a `Record<string,Endpoint>` (as per stack spec discussion)?

Reply:
Hmmm... would the key be `${project}/${region}/${id}`? The stack spec discussions inadvertently presumed that the ID is a unique key, which it isn't.

Reply:
Hmm yeah my assumption is that Spec would be both project and region locked. In this case would there ever be multiple projects at the same time? Location maybe, but not project...